hack/markdownlint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+
+if [ "${IS_CONTAINER}" != "false" ]; then
+  TOP_DIR="${1:-.}"
+  find "${TOP_DIR}" -type d \( -path ./vendor -o -path ./.github \) -prune -o -name '*.md' -exec mdl --style all --warnings {} \+
+else
+  "${CONTAINER_RUNTIME}" run --rm \
+    --env IS_CONTAINER=TRUE \
+    --volume "${PWD}:/workdir:ro,z" \
+    --entrypoint sh \
+    --workdir /workdir \
+    docker.io/pipelinecomponents/markdownlint:0.12.0@sha256:0b8f9fcf0410257b2f3548f67ffe25934cfc9877a618b9f85afcf345a25804a2 \
```

Comment:
is there some indention issue or I am hallucinating ? may be line 17 can be removed also ?

Reply:
I don't see any issue what you highlighted, but in general, the file had bad indentation carrying over from CAPM3. Fixed now.

Reply:
yeah it seems correct here, it github review view it was something like this 
![Screenshot 2023-04-03 at 13 35 32](https://user-images.githubusercontent.com/52313821/229487362-b2e9093f-9a62-4d6c-a7e3-daa15d96cdb1.png)

Its probably because of the line length the view was distorted